### PR TITLE
Release/1.0.2.0

### PR DESCRIPTION
**Pull request by P2P-Nathan**
This is the functioning code for the webinar release

### CODE REVIEW

File: Community.ManagementPackCatalog.UI/WpfElements/MarkdownViewer.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        /// <summary>
+        /// Render the provided markdown text in the window.
+        /// </summary>
+        /// <param name="markdownText">MarkDown text to display</param>
+        public void DisplayMarkdown(string markdownText)
+        {
+            this.markdownText = markdownText;
+            string htmlToRender = Markdig.Markdown.ToHtml(markdownText);
```
Comment (clarkd) [inline Community.ManagementPackCatalog.UI/WpfElements/MarkdownViewer.xaml.cs:43]: Markdig is awesome :)

File: Community.ManagementPackCatalog/CheckForManagementPackUpdates.ps1 [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        $PackDetails = Invoke-RestMethod -Uri $detailsJsonFileLocation -Method Get | Select ManagementPackSystemName,@{n="Version"; e={[version]::Parse($_.Version)}},ManagementPackDisplayName
+        
+		# Pull the management pack via SDK, with $null as the fall back
+		$InstalledMp = $null
+        $InstalledMp = Get-SCOMManagementPack -Name $PackDetails.ManagementPackSystemName
+
+		# If the pack is installed, compare the version against GitHub.  For packs with updates available, add to the OutdatedPacks counter.
+		If($InstalledMp -ne $null)
+            {
+                If($PackDetails.Version -gt $InstalledMp.Version)
```
Comment (clarkd) [inline Community.ManagementPackCatalog/CheckForManagementPackUpdates.ps1:47]: Will this do a proper version comparison?

**Review by clarkd — APPROVED** [review]

File: Community.ManagementPackCatalog.UI/WpfElements/MarkdownViewer.xaml.cs (hunk 1 above)
Comment (P2P-Nathan) [reply Community.ManagementPackCatalog.UI/WpfElements/MarkdownViewer.xaml.cs:43]: I added an inline CSS to modernize the look and have had great results with it.

File: Community.ManagementPackCatalog/CheckForManagementPackUpdates.ps1 (hunk 2 above)
Comment (P2P-Nathan) [reply Community.ManagementPackCatalog/CheckForManagementPackUpdates.ps1:47]: The PowerShell version comparison works great, I tried multiple variations using code similar to below and always received accurate results.

[version]$v1 = [version]::Parse("1.0.1.19")
[version]$v2 = [version]::Parse("1.1.0.3")
$v1 -le $v2

Comment (clarkd) [reply Community.ManagementPackCatalog/CheckForManagementPackUpdates.ps1:47]: Ah hadn't noticed you were just hooking into System.Version.Parse()

File: Community.ManagementPackCatalog.UI/WpfElements/MarkdownViewer.xaml.cs (hunk 1 above)
Comment (clarkd) [reply Community.ManagementPackCatalog.UI/WpfElements/MarkdownViewer.xaml.cs:43]: Yeah, we use it for our internal docs platform and it's awesome. We've even built out a few additional extensions for it :)

**Pull request merged**